Magic/The Treaty on Magic.md

```diff
@@ -0,0 +1,31 @@
+
+> Years later, he was a powerful wizard and showed a great determination to free himself from the oppressors. He started a war  and first attacked the guards of small villages. Therewith, his fame grew and he brought together many warriors. Thanks to his amazing powers, the One Empire of Rising Moon lost many soldiers. The difficult fields of the region and the tough climate eventually led them to fall back, abandoning their holds. Arkas self-proclaimed himself King and became revered by his people, who saw him as their new god. [..] Afterwards, some wizards among the Nendal's folk reached the level to ressurect themselves. They can't perform such magic without being able to enter the praeter in the direct way, but we don't call them archmages. Ressurection is actually forbidden in our lands thus we think the soul of the newborn must be sacrificed to let the other one come. Therefore we use to call these archmages _Witchers_ for their occult practice. They seem to gain more power with each ressurections.
+
+> The people of Nendal actually believe that Arkas and the other witchers are gods that initially live in the praeter. They think that some of them decided to come on earth and, as they are gods, their soul can't die and they can come back as newborns as they please. What is actually funny is that most of their witchers believe they are gods, simply assuming that after many ressurections, they just forget their time in the praeter among the other gods. [...] The Nendal's folk also believe that those who can perform magic in a lesser way than their gods are just graciously gifted for their religious involvement. They call them _Priests_.
+
+> At the time I'm writing this, the Nendal's folk is ruled by a theocratic government with Arkas as the spiritual leader, and where the revered witchers occupy high functions. The people are considered inferiors to the witchers and the king. Arkas even adopt the title _King of Gods_. When he reborns and is still in young age, a provisional council is formed to rule. The witchers are very heterogenous. Some of them enjoy their powers and status and don't bother with the religious principles, while others are very involved and very strict about it. Although it seems there are plans to usurp the throne, most of them are loyal to the king. Neverthless, some of them are just not interested in politic and just focus on other things like magical learning.
+
```

Comment:
I really like this nendal story. Resurection is a powerfull thing, it think that Arkas should be one of the only to have performed this.